Okta.AspNetCore/OktaAuthenticationOptionsExtensions.cs

```diff
@@ -62,11 +64,27 @@ private static AuthenticationBuilder AddCodeFlow(AuthenticationBuilder builder,
                     ValidAudience = options.ClientId,
                     NameClaimType = "name",
                 };
+
+                oidcOptions.Events.OnRedirectToIdentityProvider = BeforeRedirectToIdentityProviderAsync;
```

Comment:
I just noticed: where is the code for the logout case? Does logout work in Okta.AspNetCore 1.0.0?

Reply:
It is not required in ASP.NET Core.

Reply:
Confirmed that `id_token_hint` is sent out in ASP.NET Core